sources/lib/api/gcp/bigquery/__init__.py

```diff
@@ -12,7 +12,9 @@
 
 """Google Cloud Platform library - BigQuery Functionality."""
 
+from _csv_options import CSVOptions
```

Comment:
I can't remember specifically, but is there some guideline in python about CSVOptions vs. CsvOptions?